src/ripple/basics/strHex.h

```diff
@@ -25,6 +25,7 @@
 #ifndef RIPPLE_BASICS_STRHEX_H_INCLUDED
 #define RIPPLE_BASICS_STRHEX_H_INCLUDED
 
+#include <cassert>
```

Comment:
I always get suspicious when I see `#include`s added with no other code changes to that file.